src/test/java/org/jitsi/meet/test/web/WebParticipantOptions.java

```diff
@@ -42,10 +50,80 @@
 
     private static final String PROP_FAKE_VIDEO = "fakeStreamVideoFile";
 
+    /**
+     * The property to change remote selenium grid URL, defaults to
+     * {@link #DEFAULT_REMOTE_ADDRESS_NAME} if requiring remote browser and
+     * property is not set.
+     */
+    private static final String PROP_REMOTE_ADDRESS_NAME = "remote.address";
+
     private static final String PROP_REMOTE = "isRemote";
 
+    /**
+     * The property to evaluate for parent path of the resources used when
+     * loading chrome remotely like audio/video files.
+     */
+    private static final String PROP_REMOTE_RESOURCE_PARENT_PATH_NAME
+        = "remote.resource.path";
+
+    /**
+     * The property to disable no-sandbox parameter for chrome.
+     */
+    private static final String PROP_DISABLE_NOSANBOX
+        = "chrome.disable.nosanbox";
```

Comment:
I'm guessing the typo ("sanbox") has been there all along?

Reply:
thanks

Reply:
Yes, it was there and I will be keeping it as is with a comment. Probably it would not be a good idea to rename it now given that it will be deprecated by the new property name discussed below.